core/src/main/java/com/eaglegenomics/simlims/core/Note.java

```diff
@@ -160,4 +160,10 @@ else if (getText() != null && o.getText() != null) {
     }
     return 0;
   }
+// TODO remove me after sonarcloud does its thing
+public void junkMethodIsJunk(String unusedParam) throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "unusedParam". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)